src/components/hook/useThrottle.ts

```diff
@@ -0,0 +1,17 @@
+import { useRef, useEffect, useCallback } from 'react';
+import useStable from '@/components/hook/useStable';
+
+export default function useThrottle(fn: () => void, delay: number = 1000) {
```

Comment:
```suggestion
export default function useThrottle(fn: () => void, delay = 1000) {
```
기본값을 통해 타입이 추론되서, 이럴 때는 타입을 정의해주지 않아도 무방하답니다!